business_requirement_deliverable_project_task_categ/models/business_requirement_resource.py

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
-# © 2016 Elico Corp (www.elico-corp.com).
+# © 2017 Elico Corp (www.elico-corp.com).
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import api, fields, models
+from odoo import api, fields, models
 
 
 class BusinessRequirementDeliverableCateg(models.Model):
     _inherit = "business.requirement.resource"
 
     categ_id = fields.Many2one(
-        'project.category.main',
+        'project.tags',
```

Comment:
@YogeshMahera-SerpentCS Please use this reference and update related:

https://github.com/OCA/project/blob/10.0/project_task_category/models/project_task.py#L14